AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSPendingOperation.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+@interface MSPendingOperation : NSObject
+
+/**
+ * Pending operation.
+ */
+@property(nonatomic, strong) NSString *operation;
```

Comment:
Please use `copy` attribute for all `NSString` properties.

```suggestion
@property(nonatomic, copy) NSString *operation;
```

Reply:
@dhei can you please explain?  we don't need a deep copy here

Reply:
Yeah it's an Objective-C convention for NSString properties, you almost always want to use `copy` attribute because of immutability. We follow this convention everywhere in this repo. See more about this convention at https://stackoverflow.com/a/10715145/873234.